specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/applicationWhitelistings.json

```diff
@@ -0,0 +1,406 @@
+paths:
+  '/subscriptions/{subscriptionId}/providers/Microsoft.Security/applicationWhitelistings':
+    get:
+      description: Gets a list of application control groups for the subscription.
+      tags:
+        - applicationWhitelistings
+      parameters:
+        - $ref: '#/parameters/subscriptionId'
+        - $ref: '#/parameters/apiVersion'
+        - $ref: '#/parameters/includePathRecommendations'
+        - $ref: '#/parameters/summary'
+      responses:
+        '200':
+          description: OK
+          schema:
+            $ref: '#/definitions/AppWhitelistingGroups'
+        '400':
+          description: Bad Request - The subscription is not permitted
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+  '/subscriptions/{subscriptionId}/providers/Microsoft.Security/locations/{ascLocation}/applicationWhitelistings/{groupName}':
+    get:
+      description: Gets an application control group.
+      tags:
+        - applicationWhitelistings
+      parameters:
+        - $ref: '#/parameters/subscriptionId'
+        - $ref: '#/parameters/ascLocation'
+        - $ref: '#/parameters/groupName'
+        - $ref: '#/parameters/apiVersion'
+      responses:
+        '200':
+          description: OK
+          schema:
+            $ref: '#/definitions/AppWhitelistingGroup'
+        '400':
```

Comment:
Usually don't define negative responses and define 'default' response for all negative responses instead. Please refer to [here](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/creating-swagger.md#default-response).